libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapboxRouteLayerProviderFactory.kt

```diff
@@ -1,12 +1,12 @@
-package com.mapbox.navigation.ui.internal.route
+package com.mapbox.navigation.ui.route
 
-import com.mapbox.navigation.ui.route.RouteStyleDescriptor
+import com.mapbox.navigation.ui.internal.route.MapboxRouteLayerProvider
```

Comment:
Can `MapboxRouteLayerProvider` be move out of the `internal` package and have the `internal` Kotlin modifier too?